source/features/align-issue-labels.css

```diff
@@ -1,36 +1,22 @@
-html:not([rgh-OFF-align-issue-labels]) .js-issue-row {
```

Comment:
The old view is still used on PRs so most likely we need to preserve the old CSS too. 

You can keep the `addCssFeature` call in JS alongside the regular `add`

Reply:
https://github.com/refined-github/refined-github/pulls

Reply:
You can delete these lines, they're no longer applicable:

https://github.com/refined-github/refined-github/blob/037fa9a507b83d1cf94a71c318d777551c9ca9df/build/features.test.ts#L165-L168